trompace/mutations/musiccomposition.py

```diff
@@ -0,0 +1,275 @@
+# Generate GraphQL queries for mutations pertaining to music composition objects.
+from trompace.exceptions import UnsupportedLanguageException, NotAMimeTypeException
+from . import StringConstant
+from .templates import mutation_create, mutation_delete, mutation_link
+from ..constants import SUPPORTED_LANGUAGES
+
+ADD_MUSIC_COMPOSITION_BROAD_MATCH = '''AddMusicCompositionBroadMatch(
+    from: {{identifier: "{identifier_1}" }}
+    to: {{identifier: "{identifier_2}" }}
+  ) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+  }}'''
+
+REMOVE_MUSIC_COMPOSITION_BROAD_MATCH = '''RemoveMusicCompositionBroadMatch(
+    from: {{identifier: "{identifier_1}" }}
+    to: {{identifier: "{identifier_2}" }}
+  ) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+  }}'''
+
+ADD_MUSIC_COMPOSITION_WORK_EXAMPLE_COMPOSITION = '''AddDMusicCompositionExampleOfWork(
+    from: {{identifier: "{identifier_1}"}}
+    to: {{identifier: "{identifier_2}"}}
+) {{
+    from {{
+      identifier
+    }}
+    to {{
+      identifier
+    }}
+}}'''
+
+MERGE_MUSIC_COMPOSITION_WORK_EXAMPLE_COMPOSITION = '''MergeMusicCompositionExampleOfWork(
```

Comment:
I believe that the behaviour here is that `Merge*` methods will add it if it doesn't exist, and do nothing if it does exist. If this is the case, we don't need the `Add*` version. Can you test this on the CE and remove Add if so?